templates/empty.rc

```diff
@@ -0,0 +1,80 @@
+Magic 271485
```

Comment:
Is this file supposed to be checked in? I don't see empty.rc actually referenced anywhere.

Reply:
I believe the empty wave configuration files are referenced here https://github.com/Origen-SDK/origen_sim/blob/master/lib/origen_sim/simulator.rb#L251

Reply:
I thought earlier that the simulator.rb referenced it, then on second glace looks like they are referenced per vendor configuration and for synopsys, dve is the default viewer with extension tcl. 

I think it should be okay to remove this file.